scene/gui/control.cpp

```diff
@@ -1568,7 +1568,7 @@ void Control::set_custom_minimum_size(const Size2 &p_custom) {
 	if (p_custom == data.custom_minimum_size) {
 		return;
 	}
-	data.custom_minimum_size = p_custom;
+	data.custom_minimum_size = p_custom.round();
```

Comment:
I still think that this method (and `set_fixed_icon_size` I guess) should take `Size2i` instead, then it's the responsibility of the caller to make sure the size provided is rounded the way they want it. `popup_centered` already takes `Size2i` for example.

Reply:
Sure, I just think it feels weird for just that `Control` function to accept `Size2i` while the others are still `Size2`.

Reply:
Gotta start somewhere :) Also controls are pixel perfect, don't have any sub-pixels, right? So it makes sense here specifically.

Reply:
Done.